cmd/aries-js-worker/main.go

```diff
@@ -66,6 +84,41 @@ func main() {
 	select {}
 }
 
+// initAries creates aries agent instance
+// TODO currently running on default opts, should be able pass custom framework opts
+func getAllCommands() []cmdctrl.Handler {
+	msgHandler := msghandler.NewRegistrar()
+
+	a, err := aries.New(aries.WithMessageServiceProvider(msgHandler))
+	if err != nil {
+		js.Global().Get("console").Call("error",
```

Comment:
This should be an event to the JS wrapper rather than hard coding the console call.

Reply:
Will be added as part of upcoming PR where we are going to initialize aries framework using opts and any failure will be notified during loading.